src/display/annotation_layer.js

```diff
@@ -816,9 +824,10 @@ class LinkAnnotationElement extends AnnotationElement {
    * @private
    * @param {Object} link
    * @param {Object} destination
+   * @param {str} [overlaidText]
```

Comment:
s/str/string here and for all instances added below (see e.g. https://github.com/calixteman/pdf.js/blob/8c59909ed6ab83e13f22ba981d42754d0a538219/src/display/api.js#L237)